llvm/test/CodeGen/RISCV/GlobalISel/instruction-select/bswap-rv32.mir

```diff
@@ -2,6 +2,9 @@
 # RUN: llc -mtriple=riscv32 -mattr=+zbb -run-pass=instruction-select \
 # RUN:   -simplify-mir -verify-machineinstrs %s -o - \
 # RUN:   | FileCheck -check-prefix=RV32I %s
+# RUN: llc -mtriple=riscv32 -mattr=+zbkb -run-pass=instruction-select \
+# RUN:   -simplify-mir -verify-machineinstrs %s -o - \
+# RUN:   | FileCheck -check-prefix=RV32I %s
```

Comment:
Should we change the prefix here and in 64-bit version also to `RV...ZBB_OR_ZBKB`?

Reply:
Sure. Or we could just use the default `CHECK`

Reply:
changed to "CHECK"